.gitlab-ci.yml

```diff
@@ -199,8 +198,8 @@ default:
     - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/ # PRs
     - if: $CI_COMMIT_REF_NAME =~ /^gh-readonly-queue.*$/ # merge queues
 
-.zombienet-refs:
-  extends: .build-refs
+# .zombienet-refs:
```

Comment:
This is intentional here?

Reply:
Yes, I'm checking moving the version out of the main file to not require approves  from release-eng. I will remove it and ping you when is ready. 
cc: @bkchr